### DISCUSSION TITLE
What is the difference between ActiveSelection and Group?

### DISCUSSION THREAD

**Discussion by vaynevayne** [fabricjs/fabric.js#10103] [Q&A]
I want a group of rectangles to move, scale, rotate, and get the coordinates of the rectangles after operation.
When I use Group scaling, I need to calculate the rect. left top width based on the group.scale and the Group center point, which is very troublesome.
I wonder if using ActiveSelection can simplify this process?

**Comment by asturur** [comment]
No is unrelated.
ActiveSelection is a tool of the canvas to maintain active selection.

You have to look in the transformPoint utilty and the obect's calcTransformMatrix method and learn about sending a point from plane to another.

There are utilities that simplify the process overall.

You could also look at getCoords method of the rectangles that can be useful too
